lib/controller.rb

```diff
@@ -54,10 +54,15 @@ class SlowFood < Sinatra::Base
     erb :index
   end
 
-  get '/auth/login' do
-    erb :login
+  get '/menu' do
+    @dishes = Dish.all
+    erb :menu
   end
 
+  # get '/auth/login' do
+  #   erb :login
+  # end
+
```

Comment:
If you won't be using this method, please remove it. Not cool to keep comments around.